materialize.html

```diff
@@ -35,11 +35,19 @@ <h2>About Us</h2>
                <p>Freddy's Pizzeria was founded in 1987 by William B. Afton. Our first location opened in Hurricane, Utah, and has since expanded across the state, bringing fun animatronics and great food to families everywhere.</p>
            </div>
            <div class="col s12 m6">
-               <img src="images/Pizzzera.jpg" alt="Freddy's Pizzeria" class="responsive-img">
+               <img src="images/Pizzzera.jpg" class="circle responsive-img" alt="Freddy's Pizzeria"  width="200" height="200" class="responsive-img">
```

Comment:
@AaronA951 Why do you have two class attributes here? You can delete the one at the end since it's repetitive.